README.md

```diff
@@ -11,10 +13,24 @@
 - 📝 Line-specific comments on code changes
 - 🤖 Auto-fix suggestions with new PRs
 - 🎯 Project-specific guidelines
+- 🚫 Skip review functionality with special PR titles
```

Comment:
✅ Nice addition of the skip review functionality! This provides users with more control over their PRs. 🎉